cli_tools/osconfig_agent/patch_window_test.go

```diff
@@ -20,6 +20,7 @@ import (
 
 	osconfigpb "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/osconfig_agent/_internal/gapi-cloud-osconfig-go/google.golang.org/genproto/googleapis/cloud/osconfig/v1alpha1"
 	"github.com/golang/protobuf/ptypes/duration"
+	date "google.golang.org/genproto/googleapis/type/date"
```

Comment:
no need for the named import, just "google.golang.org/genproto/googleapis/type/date"

I'll have to go back through and rename all the pb imports to NAMEpb as is standard